gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/users/UsersResource.java

```diff
@@ -105,22 +113,10 @@ public void create(
             @Suspended final AsyncResponse response) {
         final io.gravitee.am.identityprovider.api.User authenticatedUser = getAuthenticatedUser();
 
-        // user must have a password in no pre registration mode
-        if (!newUser.isPreRegistration() && newUser.getPassword() == null) {
-            response.resume(new UserInvalidException(("Field [password] is required")));
-            return;
-        }
-
-        // check password policy
-        if (newUser.getPassword() != null) {
-            if (!passwordValidator.validate(newUser.getPassword())) {
-                response.resume(new UserInvalidException(("Field [password] is invalid")));
-                return;
-            }
-        }
-
         checkPermission(ReferenceType.ORGANIZATION, organizationId, Permission.ORGANIZATION_USER, Acl.CREATE)
-                .andThen(userService.create(ReferenceType.ORGANIZATION, organizationId, newUser, authenticatedUser)
+                .andThen(domainService.findById(organizationId)
```

Comment:
This resource is for the organization. Nothing related to domain here (i.e domainService.findById() will always failed)

Reply:
ok